<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reversi</string>
  </property>
  <widget class="QWidget" name="MainMenu" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>301</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="menu">
    <item>
     <widget class="QLabel" name="Title_Label">
      <property name="styleSheet">
       <string notr="true">font: 40pt &quot;Adobe Arabic&quot;;
color:rgb(255, 170, 255)</string>
      </property>
      <property name="text">
       <string>^_^</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Reversi_Button">
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>Reversi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="FIR_Button">
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>5inaRow</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Go_Button">
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Board">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>480</width>
     <height>479</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/new/images/chessboard.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="GameMenu" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>161</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Gamemenu" rowstretch="1,2,2,2,2,2">
    <property name="verticalSpacing">
     <number>30</number>
    </property>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="Menu_Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="Undo_Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="Start_Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLCDNumber" name="BCount_LCD">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="WCount_LCD">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="WCount_Label">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>White</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="BCount_Label">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>Black</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Save_Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Load_Button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
